sdk/communication/CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributing Guide
+
+This a contributing guide made specifically for the Azure Communication Services SDK. The Azure SDK repo also has a contributing guide that might help you in some other general processes this guide assumes you have done. If you haven't checked that one out yet, you can find it [here](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md)
+
+The Azure Communication Services SDK for JS currently consists of six different packages. While each package has its own set of environment variables to make their tests run successfully, all of them follow a similar structure that allows a smooth onboarding process.
+
+Let's get started with how to set up the repo itself.
+
+## Building
+
+Install and link all dependencies `rush update`, and build the code base `rush rebuild`.
+You can then `rush build -t <packagename>` to build a single project.
+Once the package has been built, let's jump on how to run the tests to see that everything is in order.
+
+## Testing
+
+Make sure to check out the general contributing guide the Azure SDK repo has for a more in-depth look at testing and setting up your dev environment. You can check out the contributing file [here](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md)
+
+When you go inside the package you are working with, you will see a folders called `recordings\browsers` and `recordings\node`. In JS we performs tests in the node and browser environment. Each folder contains, as its name suggests, recordings of successful calls to the API that allow us to run the tests in PLAYBACK mode and remove the necessity of hitting the actual resources every time we may want to test.
+
+To test with environment variables, JS has a `sample.env` for developers to use as a template. You can rename the file to `.env` and add the appropriate environment variables.
```

Comment:
Don't think we should be advising a rename of `sample.env` or `test.env` because people might check in the change and these templates will get deleted.

Reply:
I believe the `.env` is already being git ignored but you are right they can accidentally push up a deleted file. So I think I will re-word it instead of having them rename the file I will say: "Create your own `.env` using `sample.env` as reference"

Reply:
Thoughts?